<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunReportDialog</class>
 <widget class="QDialog" name="RunReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="report_picker">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTreeWidget" name="treeWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
         <widget class="QTextBrowser" name="textBrowser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="timespan_picker">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="lblReportTitle">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Report&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>318</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="rbToday">
         <property name="text">
          <string>Today</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="rbYesterday">
         <property name="text">
          <string>Yesterday</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QRadioButton" name="rbDateSpan">
         <property name="text">
          <string>Date Span:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QDateEdit" name="deEndDate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QDateEdit" name="deStartDate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="rbThisMonth">
         <property name="text">
          <string>This Month</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QRadioButton" name="rbLastYear">
         <property name="text">
          <string>Last Year</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QRadioButton" name="rbTimeSpan">
         <property name="text">
          <string>Time Span:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Choose the timespan to run the report over:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>to</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QRadioButton" name="rbLastWeek">
         <property name="text">
          <string>Last Week</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="rbThisWeek">
         <property name="text">
          <string>This Week</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="rbThisYear">
         <property name="text">
          <string>This Year</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="rbLastMonth">
         <property name="text">
          <string>Last Month</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>to</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QDateTimeEdit" name="teStartTime">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QDateTimeEdit" name="teEndTime">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QRadioButton" name="rbMonth">
         <property name="text">
          <string>Month:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDateEdit" name="deMonth">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::MonthSection</enum>
         </property>
         <property name="displayFormat">
          <string>MM/yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="rbDate">
         <property name="text">
          <string>Date:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDateEdit" name="deDate">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QRadioButton" name="rbYear">
         <property name="text">
          <string>Year:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QDateEdit" name="deYear">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="displayFormat">
          <string>yyyy</string>
         </property>
         <property name="calendarPopup">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbBack">
       <property name="text">
        <string>&lt; &amp;Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNext">
       <property name="text">
        <string>&amp;Next &gt;</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbToday</tabstop>
  <tabstop>rbYesterday</tabstop>
  <tabstop>rbDate</tabstop>
  <tabstop>deDate</tabstop>
  <tabstop>rbThisWeek</tabstop>
  <tabstop>rbLastWeek</tabstop>
  <tabstop>rbThisMonth</tabstop>
  <tabstop>rbLastMonth</tabstop>
  <tabstop>rbMonth</tabstop>
  <tabstop>deMonth</tabstop>
  <tabstop>rbThisYear</tabstop>
  <tabstop>rbLastYear</tabstop>
  <tabstop>rbYear</tabstop>
  <tabstop>deYear</tabstop>
  <tabstop>rbDateSpan</tabstop>
  <tabstop>deStartDate</tabstop>
  <tabstop>deEndDate</tabstop>
  <tabstop>rbTimeSpan</tabstop>
  <tabstop>teStartTime</tabstop>
  <tabstop>teEndTime</tabstop>
  <tabstop>pbBack</tabstop>
  <tabstop>pbNext</tabstop>
  <tabstop>pbCancel</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>rbDate</sender>
   <signal>toggled(bool)</signal>
   <receiver>deDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbMonth</sender>
   <signal>toggled(bool)</signal>
   <receiver>deMonth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbYear</sender>
   <signal>toggled(bool)</signal>
   <receiver>deYear</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDateSpan</sender>
   <signal>toggled(bool)</signal>
   <receiver>deStartDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbDateSpan</sender>
   <signal>toggled(bool)</signal>
   <receiver>deEndDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbTimeSpan</sender>
   <signal>toggled(bool)</signal>
   <receiver>teStartTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbTimeSpan</sender>
   <signal>toggled(bool)</signal>
   <receiver>teEndTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
